docker-compose.yml

```diff
@@ -1,26 +1,41 @@
-version: '2.2'
+version: '3.7'
 services:
   couch:
-    image: couchdb:2.1.1
-    ports:
-      - ${COUCH_PORT}:5984
+    image: couchdb:2.3.1
     environment:
-      COUCHDB_USER: ${COUCH_ADMIN}
-      COUCHDB_PASSWORD: ${COUCH_PASS}
+      COUCHDB_USER: cht
+      COUCHDB_PASSWORD: cht-password
+    networks:
+      - cht-net
   postgres:
-    image: postgres:9.4
-    ports:
-      - ${PG_PORT}:5432
```

Comment:
let's expose this port again to resolve Nick's comment above. If there's a reason not to, let's reply and add some documentation to the README on how to expose if required.